client/src/pages/auth.tsx

```diff
@@ -89,6 +97,16 @@ class Auth extends React.Component<ISignupPageProps> {
                                 width: 400,
```

Comment:
Since we're no longer using our own login system we can probably delete the text fields for email/username/password